docs/get-started.md

```diff
@@ -47,21 +23,20 @@ Instead of direct and synchronous interactions between components, producing and
 Your system has **delayed or no user notifications**:<br/>
 Although there are many systems that would benefit from events, almost any scenario where a side effect of the business event is “to notify the Veteran” could be a candidate for an event stream.
 
-## The Enterprise Event Bus Is Not a Good Fit If ...
+## The Enterprise Event Bus is not a good fit if ...
 
-You are interested in a **one-time data sweep**<br/>
+You are interested in a **one-time data sweep**:<br/>
 If you just need to search through a data source to identify specific cases for further processing, event-driven systems would introduce unnecessary complexity and overhead. The Enterprise Event Bus is geared towards handling ongoing event streams rather than singular, one-time data analysis tasks.
 
-Your system deals with **simple, linear workflows**<br/>
+Your system deals with **simple, linear workflows**:<br/>
 If you are dealing with a system that has no significant interaction between components, the overhead of event handling might outweigh the benefits.
 
-Your system is based on a **different programming paradigm**<br/>
+Your system is based on a **different programming paradigm**:<br/>
 There are many valid reasons to use other architectures, such as synchronous API-based technologies. Event-driven architecture is not the best solution in every scenario.
 
-Your system **lacks well-defined components, or has constantly changing interactions**<br/>
+Your system **lacks well-defined components, or has constantly changing interactions**:<br/>
 Systems that are continuously in flux, or don't have well-established boundaries or communication patters would make it difficult to introduce event-driven designs.
 
-## Reach Out To Us
-If you think the Enterprise Event Bus would be a good fit for your situation, or help you solve a problem, then please get in touch with us to discuss further.
+Your system **is rated as “high” under [FISMA](https://security.cms.gov/learn/federal-information-security-management-act-fisma) and the [VA system categorization](https://jubilant-succotash-m55rqe7.pages.github.io/categorization/)**:<br/> 
+The Event Bus is hosted on the LHDI platform. Because the platform is rated at a “Medium” level under FISMA and the VA system categorization, we are only accepting applications that are rated as Low or Moderate systems at this time.”
 
-The best way to reach out to the Event Bus Team is via the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) Slack channel in the Digital Service VA workspace.
```

Comment:
It looks like the "Reach out to us" section that's already on this portal page didn't get added to the Google doc. I think it's a good way to conclude the page though, so let's keep it. The only change needed is to:
1. Change the header ("Reach out to us") to be sentence case, and 
2. Replace "Digital Service VA" with "Office of CTO @VA" (which is how it's displayed on Slack).

Reply:
This is done. Marking as resolved.